app/apps/server/bridges/scheduler.js

```diff
@@ -2,22 +2,42 @@ import Agenda from 'agenda';
 import { MongoInternals } from 'meteor/mongo';
 
 function _callProcessor(processor) {
-	return (job) => processor(job.attrs.data);
+	return (job) => processor(job);
```

Comment:
IMO we should not send Agenda's object directly to the app - this will make us directly dependant on Agenda's public API and might expose the host process to the app.

If we want to provide a more flexible object to the processor in the app, the engine should create such an object and call the appropriate bridges.

Reply:
Solved. Now the processor only receives the `data` object (or an empty one). I think this solves it